prototype-business-case-joined-up-services.MD

```diff
@@ -1,138 +1,79 @@
 # [Prototype] Business case to help join up government services
 
+References in brackets are the [key decisions and evidence](https://github.com/alphagov/prototype-case-joined-up-services/blob/master/key-decisions-and-evidence.MD) that supports this business case.
+
 ## Strategic Case
 
 ### Strategic Context
 
-We aim to provide an initial case for investment to help Service Design and Technology standards deliver on the 2030 Vision for the DDaT function.
+Service Design and Technology Standards at the Government Digital Service (GDS) sets the bar for what good services look like, and offers support to guide people using technology to improve services. We're making an initial case for investment to help  deliver on the 2030 Vision for the Digital, Data, and Technology (DDaT) function.
 
-The vision says, "in 2030, government services will be joined up, trusted and responsive to user needs." GDS will help the DDaT function deliver on the promises in the vision. GDS Service Design and Technology Standards sets the bar for what good services look like. We want to provide better support for government to meet those standards.
+The vision says, "in 2030, government will be joined up, trusted and responsive to user needs."
 
-This case for investment aims to compliment work on GOV.UK. We will make incremental changes to this prototype business case, informed by user research and data analysis.
+With other parts of GDS focusing on 'trusted' and 'responsive to user needs', we're focusing on helping government become more joined up. Currently, government services are not very joined up:
```

Comment:
Add a paragraph around what this business case is for. It is about getting money for starting service communities again or to fund our team to continue doing discoveries...

Reply:
Added to my branch. Thanks